### ISSUE TITLE
Website Accessibility-WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (a[href$="foundation\/"])

### ISSUE THREAD

**Issue by A11Ybot-OP** [matrix-org/matrix.org#1163]
#### Issue
`Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds` ([color-contrast](https://accessibilityinsights.io/info-examples/web/color-contrast))
#### Target application
[Matrix.org](https://matrix.org/)
#### Element path
.mxmatrixdescription.mxp:nth-child(3) > a[href$="foundation\/"]
#### Snippet
`<a href="/foundation/">Matrix.org Foundation</a>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 3.25 (foreground color: #0098d4, background color: #ffffff, font size: 15.0pt (20px), font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Microsoft Edge version 96.0.1054.62
